dozer-api/src/grpc/types_helper.rs

```diff
@@ -108,8 +110,9 @@ fn field_to_prost_value(f: Field) -> Value {
         },
         Field::Decimal(d) => map_decimal(d),
         Field::Timestamp(ts) => Value {
-            value: Some(value::Value::StringValue(
-                ts.to_rfc3339_opts(SecondsFormat::Millis, true),
+            value: Some(value::Value::TimestampValue(
+                Timestamp::from_str(&ts.to_rfc3339_opts(SecondsFormat::Millis, true))
```

Comment:
Do we have to go through `String`?

Reply:
in case we want to return this as `google.protobuf.Timestamp` then we don't need to make this as String

Reply:
I get where your comment is coming from, but we need to decide whether we want to keep the result of the fields to be

currently, with this pr's change below is the output from typed grpc

```
{
            "typ": "String",
            "name": "create_date",
            "nullable": false
        },
        {
            "typ": "Timestamp",
            "name": "last_update",
            "nullable": true
        },
```

Reply:
> decide whether we want to keep the result of the fields to be

This is the question I wanted to ask in the other comment.

This comment is because I see when we convert a `Field::Timestamp` to `grpc::Timestamp`, it goes like this `Filed::Timpstamp` -> `String` -> `grpc::Timestamp`. I was wondering if this is necessary. Can we save the formatting and parsing cost?